packages/react-ds/src/components/SplitButton/SplitButton.stories.mdx

```diff
@@ -58,6 +64,88 @@ import SplitButton from './SplitButton'
   </Story>
 </Preview>
 
+## Menu
+
+### Menu Fit Button Container
+
+<Preview>
+  <Story name="Menu Fit Button Container" height="200px">
```

Comment:
Maybe we don't need a story for it, just a `knob`, do you agree? 🤔

Reply:
Of course my friend

Reply:
Made on 8e55222